tests/test_values.py

```diff
@@ -7,16 +7,18 @@
 
 from mock import patch
 
-from configurations.values import (Value, BooleanValue, IntegerValue,
-                                   FloatValue, DecimalValue, ListValue,
-                                   TupleValue, SingleNestedTupleValue,
-                                   SingleNestedListValue, SetValue,
-                                   DictValue, URLValue, EmailValue, IPValue,
-                                   RegexValue, PathValue, SecretValue,
-                                   DatabaseURLValue, EmailURLValue,
-                                   CacheURLValue, BackendsValue,
-                                   CastingMixin, SearchURLValue,
-                                   setup_value)
+from configurations.values import (
+    Value, BooleanValue, IntegerValue,
+    FloatValue, DecimalValue, ListValue,
+    TupleValue, SingleNestedTupleValue,
+    SingleNestedListValue, SetValue,
+    DictValue, URLValue, EmailValue, IPValue,
+    RegexValue, PathValue, SecretValue,
+    DatabaseURLValue, EmailURLValue,
+    CacheURLValue, BackendsValue,
+    CastingMixin, SearchURLValue,
+    setup_value,
```

Comment:
Could be wrapped better?!  (since it is touched already).

Reply:
But seems to not have been changed actually?
We should bring isort then in a separate PR.